From 5a2e97f8dfed8f859adf0b392a177e598bc8b10d Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 26 Feb 2020 08:24:29 +0200
Subject: [PATCH 26/88] rework Zilla font

	modified:   data/fonts/fonts.xml
	modified:   packages/overlays/FontZillaOverlay/AndroidManifest.xml
	modified:   packages/overlays/FontZillaOverlay/res/values/config.xml
---
 data/fonts/fonts.xml                                     | 2 +-
 packages/overlays/FontZillaOverlay/AndroidManifest.xml   | 2 +-
 packages/overlays/FontZillaOverlay/res/values/config.xml | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/data/fonts/fonts.xml b/data/fonts/fonts.xml
index 7748e242653..87b52c435a1 100644
--- a/data/fonts/fonts.xml
+++ b/data/fonts/fonts.xml
@@ -147,7 +147,7 @@
         <font weight="600" style="normal">ZillaSlab-SemiBold.ttf</font>
         <font weight="600" style="italic">ZillaSlab-SemiBoldItalic.ttf</font>
     </family>
-    <alias name="zilla-slab-medium" to="zilla-slab" weight="500" />
+    <alias name="zilla-slab-bold" to="zilla-slab" weight="700" />
 
     <!-- fallback fonts -->
     <family lang="und-Arab" variant="elegant">
diff --git a/packages/overlays/FontZillaOverlay/AndroidManifest.xml b/packages/overlays/FontZillaOverlay/AndroidManifest.xml
index 994a87fb578..12cd789ecd9 100644
--- a/packages/overlays/FontZillaOverlay/AndroidManifest.xml
+++ b/packages/overlays/FontZillaOverlay/AndroidManifest.xml
@@ -26,6 +26,6 @@
     <application android:label="@string/font_zilla_overlay" android:hasCode="false">
         <meta-data
             android:name="android.theme.customization.REQUIRED_SYSTEM_FONTS"
-            android:value="zilla-slab,zilla-slab-medium" />
+            android:value="zilla-slab,zilla-slab-bold" />
     </application>
 </manifest>
diff --git a/packages/overlays/FontZillaOverlay/res/values/config.xml b/packages/overlays/FontZillaOverlay/res/values/config.xml
index a7308429fe2..dfe37199301 100644
--- a/packages/overlays/FontZillaOverlay/res/values/config.xml
+++ b/packages/overlays/FontZillaOverlay/res/values/config.xml
@@ -19,9 +19,9 @@
      <!-- Name of a font family to use for body text. -->
     <string name="config_bodyFontFamily" translatable="false">zilla-slab</string>
     <!-- Name of a font family to use for medium body text. -->
-    <string name="config_bodyFontFamilyMedium" translatable="false">zilla-slab-medium</string>
+    <string name="config_bodyFontFamilyMedium" translatable="false">zilla-slab-bold</string>
     <!-- Name of a font family to use for headlines. If empty, falls back to platform default -->
     <string name="config_headlineFontFamily" translatable="false">zilla-slab</string>
     <!-- Name of the font family used for system surfaces where the font should use medium weight -->
-    <string name="config_headlineFontFamilyMedium" translatable="false">zilla-slab-medium</string>
+    <string name="config_headlineFontFamilyMedium" translatable="false">zilla-slab-bold</string>
 </resources>
-- 
2.17.1

